src/components/Form/fields.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Input from 'cozy-ui/transpiled/react/Input'
+import Textarea from 'cozy-ui/transpiled/react/Textarea'
+
+function adaptField(WrappedComponent) {
+  class AdaptField extends React.Component {
+    render() {
+      const {
+        input,
+        meta: { valid },
```

Comment:
Can we use custom attributes? Something like `data-valid` ?

Reply:
I can change it here: `<WrappedComponent {...input} {...rest} data-valid={valid} />`. Note that we don't use `react-final-form`'s validation so `meta.valid` is always `true` and `cozy-ui` does not take either `valid` or `data-valid` prop into account for now.

Reply:
That is exactly my point: I don't think having a `valid` attribute in the generated DOM is a good idea since `valid` is not a valide attribute : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes

Reply:
done, thanks!